README.md

```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/report_navigate.png" style="border-radius: 15px" alt="Status Bar"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/command_palette.png" alt="VS Code Embedded Commands for Testing"/>
 </p>
 
-- You can navigate to the source location of the target test case by clicking the navigate button.
-
-> Note: You can use `java.test.report.showAfterExecution` to configure whether to automatically show the test report after execution. By default, it will be shown when there are failed tests. 
-
+There are some other VS Code embedded commands for testing, which can be found by searching `Test:` in the Command Palette.
 
 ## Settings
 
 | Setting Name | Description | Default Value |
 |---|---|---|
-| `java.test.report.position` | Specify where to show the test report. Supported values are: `sideView`, `currentView`. | `sideView` |
-| `java.test.report.showAfterExecution` | Specify if the test report will automatically be shown after execution. Supported values are: `always`, `onFailure`, `never`. | `onFailure` |
-| `java.test.editor.enableShortcuts` | Specify whether to show the Code Lenses in editor or not. | `true` |
-| `java.test.log.level` | Specify the level of the test logs. Supported values are: `error`, `info`, `verbose`. | `info` |
 | `java.test.config` | Specify the configuration for the test cases to run with. [More details](https://aka.ms/java-test-config). | `{}` |
 | `java.test.defaultConfig` | Specify the name of the default test configuration. | `""` |
 
+### VS Code Embedded Settings for Testing
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/settings.png" alt="VS Code Embedded Settings for Testing"/>
+</p>
+
+There are some other VS Code embedded settings for testing, which can be found by searching `testing` in the Settings view.
+## Project Setup
+### JUnit 5
+
+Please refer to [Getting Started](https://junit.org/junit5/docs/current/user-guide/#overview-getting-started) from the JUnit 5's official document for getting started guide.
+
+> Note: If your project does not use build tools(Maven/Gradle/...), please make sure [junit-platform-console-standalone.jar](https://search.maven.org/search?q=g:org.junit.platform%20AND%20a:junit-platform-console-standalone) is on your project classpath.
+
+### JUnit 4
+Please refer to [Download and Install](https://github.com/junit-team/junit4/wiki/Download-and-Install) from the JUnit 4's official document for the getting started guide.
+
+### TestNG
+Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's official document for getting started guide.
+
 ## FAQ
 If you meet any problem when using the extension, please refer to the [FAQ](https://github.com/microsoft/vscode-java-test/wiki/FAQ) to check if there is an answer to your problem.
 
```

Comment:
possible to add Github Repo Issues link?

Reply:
ok